pyquil/noise.py

```diff
@@ -394,6 +396,7 @@ def get_noisy_gate(gate_name: str, params: Iterable[ParameterDesignator]) -> Tup
     elif gate_name == "CZ":
         assert params == ()
         return np.diag([1, 1, 1, -1]), "NOISY-CZ"
+    # QUESTION: Do we want to support XY here?
```

Comment:
Can you create a "Support XY gates in noise model generation" issue for the backlog? Seems like that could happen before or after v3 release